pkg/api/validation/validation_test.go

```diff
@@ -919,7 +919,7 @@ func TestValidateEnv(t *testing.T) {
 					},
 				},
 			}},
-			expectedError: "valueFrom.fieldRef.fieldPath: unsupported value 'status.phase', Details: supported values: metadata.name, metadata.namespace, status.podIP",
+			expectedError: `valueFrom.fieldRef.fieldPath: Unsupported value: "status.phase": supported values: metadata.name, metadata.namespace, status.podIP`,
```

Comment:
It's pretty go-style - add details: at the front.

On Tue, Dec 15, 2015 at 11:42 PM, Chao Xu <notifications@github.com> wrote:

> In pkg/api/validation/validation_test.go
> <https://github.com/kubernetes/kubernetes/pull/18694#discussion_r47744682>
> :
>
> > @@ -919,7 +919,7 @@ func TestValidateEnv(t *testing.T) {
> >  					},
> >  				},
> >  			}},
> > -			expectedError: "valueFrom.fieldRef.fieldPath: unsupported value 'status.phase', Details: supported values: metadata.name, metadata.namespace, status.podIP",
> > +			expectedError: `valueFrom.fieldRef.fieldPath: Unsupported value: "status.phase": supported values: metadata.name, metadata.namespace, status.podIP`,
>
> The multiple colons in a single sentence still makes me frown. Is this
> following some tradition?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18694/files#r47744682>.
>